docs/topics/secured-mission-intro-paragraph.adoc

```diff
@@ -1,6 +1,8 @@
-Mission proficiency level: 
+Mission proficiency level:
 //special case since topic is used by front end.
-ifdef::docs-topic[xref:proficiency_foundational[*{proficiency-foundational}*].]
-ifndef::docs-topic[link:https://appdev.openshift.io/docs/wf-swarm-runtime.html#proficiency_levels[*{proficiency-foundational}*].]
+ifdef::docs-topic[xref:proficiency_advanced[*{proficiency-advanced}*].]
+ifndef::docs-topic[link:https://appdev.openshift.io/docs/wf-swarm-runtime.html#proficiency_levels[*{proficiency-advanced}*].]
 
 The Secured Booster expands on the {name-mission-http-api} Booster by securing a REST endpoint using link:https://access.redhat.com/products/red-hat-single-sign-on[{RHSSO}]. {RHSSO} implements the OAuth 2.0 specification and uses it to issue access tokens to provide clients with various access rights to secured resources. Securing an application with SSO enables you to add security to your applications while centralizing the security configuration.
+
+IMPORTANT:  You must configure {RHSSO} and Keycloak to access certain components included in the Secured Mission. This mission does not show you how to perform this configuration. Ensure that you are familiar with the basic concepts related to {RHSSO} and Keycloak before using this mission. 
```

Comment:
It's true that the mission doesn't show how to perform this configuration, but it comes with SSO preconfigured for demonstration purposes, so "You must configure {RHSSO} and Keycloak" isn't exactly accurate.

Also Keycloak is an upstream of RHSSO, so referring to "RHSSO and Keycloak" isn't exactly accurate.